doc/proxy-metrics.md

```diff
@@ -133,3 +135,63 @@ request_total{
 
 [prom-format]: https://prometheus.io/docs/instrumenting/exposition_formats/#format-version-0.0.4
 [pod-template-hash]: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#pod-template-hash-label
+
+# Transport-Level Metrics
+
+The following metrics are collected at the level of the underlying transport
+layer.
+
+### `connection_opens`
+
+A counter of the total number of transport connections which have been opened.
+
+### `connection_closes`
+
+A counter of the total number of transport connections which have been closed.
+
+### `connections`
```

Comment:
similarly, perhaps this should be `open_connections` or `active_connections` or something that disambiguates it.

Reply:
yeah, agree in principal about disambiguation. if i see a counter like `connection_opens`, i'd think `connections_open` would be most intuitive as a gauge, but not a big deal.

fwiw, Prometheus recommends suffixing counters with `_total`:
https://prometheus.io/docs/practices/naming/#metric-names

Reply:
perhaps `tcp_connections_open`